spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -993,7 +993,7 @@ def name_id_version(format_urn)
             authn_context_comparison: 'exact',
             service_provider: 'http://localhost:3000',
             request_signed: true,
-            requested_ial: 'http://idmanagement.gov/ns/assurance/loa/5',
+            requested_ial: 'none',
```

Comment:
this came up in @vrajmohan's refactor as well -- since that was a true refactor we left the bug in. 

it's only affecting the analytics, so it *should* be fine, but can you please ask in the app-dev channel to make sure no one is using that field for debugging purposes or something?

Reply:
I had already created an issue for this - https://gitlab.login.gov/lg-people/Melba/backlog-fy24/-/issues/99

Reply:
I actually don't think this is an issue. Reporting function are the only callers of `#requested_ial` and only when VOTs aren't present.

The `none` case seems to mean we weren't able to find a requested IAL since if it isn't in our list of known IAL values, how would we be able to identify it as an IAL? We also log all unfiltered ACRs in the `authn_context` field, so there is no data loss in that regard.